components/IngredientSelector/Navigator.tsx

```diff
@@ -170,7 +169,7 @@ export default function Navigator() {
           onChange={(event) =>
             dispatch(updateValue({ quantityValue: Number(event.target.value) }))
           }
-          endDecorator={isWeightValue ? <Typography>g</Typography> : null}
+          endDecorator={isWeightValue ? <Typography>g</Typography> : (quantityValue > 1 ? currentQuantity?.quantity_name : currentQuantity?.quantity_name_1)}
```

Comment:
build fails because `quantityValue` can be `null, and so can not be compared to 1.

Two solution: 

- Say to typescript that it's not null thanks to a `!`

```suggestion
          endDecorator={isWeightValue ? <Typography>g</Typography> : (quantityValue! > 1 ? currentQuantity?.quantity_name : currentQuantity?.quantity_name_1)}
```

- test it's not null
```suggestion
          endDecorator={isWeightValue ? <Typography>g</Typography> : (quantityValue && quantityValue > 1 ? currentQuantity?.quantity_name : currentQuantity?.quantity_name_1)}
```

Reply:
Thanks, I need to learn more about typescript.